src/adapters/ckeditor-4-adapter.ts

```diff
@@ -15,10 +15,10 @@
  */
 
 import { Match, MatchWithReplacement } from '@acrolinx/sidebar-interface';
-import { AbstractRichtextEditorAdapter } from './AbstractRichtextEditorAdapter';
-import { HasEditorID, ContentExtractionResult } from './AdapterInterface';
+import { AbstractRichtextEditorAdapter } from './abstract-rich-text-editor-adapter';
+import { HasEditorID, ContentExtractionResult } from './adapter-interface';
 
-export class CKEditorAdapter extends AbstractRichtextEditorAdapter {
+export class CKEditor4Adapter extends AbstractRichtextEditorAdapter {
```

Comment:
This is an exported class. Is this rename safe to do? Both the class name and in the filename.